reconcile/openshift_users.py

```diff
@@ -14,21 +15,38 @@
 QONTRACT_INTEGRATION = "openshift-users"
 
 
-def get_cluster_users(cluster, oc_map):
+def get_cluster_users(cluster, oc_map, clusters):
```

Comment:
it would be cool to have a `starmap` version of `threaded.run`, then we could do something like

```
def get_cluster_users(cluster_info, ocm_cluster):
   ...


results = threaded.run(
        get_cluster_users,
        [
          (cluster_info, ocm_cluster) ... rest of the expression that generates a list of cluster_info,ocm_cluster tuples
        ]
        thread_pool_size,
    )
```

the situation that `cluster_info` would be None (or at least the mypy required None check), could be avoided right from the beginning.

Reply:
I want to keep the current solution, it is the less intrusive